official/transformer/utils/tokenizer.py

```diff
@@ -61,11 +64,15 @@
 class Subtokenizer(object):
   """Encodes and decodes strings to/from integer IDs."""
 
-  def __init__(self, vocab_file, reserved_tokens=None):
+  def __init__(self, vocab_file, reserved_tokens=None,
+               master_char_set=None):
     """Initializes class, creating a vocab file if data_files is provided."""
```

Comment:
directly use master_char_set= _ALPHANUMERIC_CHAR_SET? Is it better

Reply:
I thought about it and finally decided to propose the `None`-style because I think it more convenient for future support:
1. It is harder to make an error when defining default arguments of downstream functions.
2. It keeps definition of `_ALPHANUMERIC_CHAR_SET` local tokenizer module. E.g. it allows hackers to tweak the global variable at runtime if they want to. AFAIK, mantioning it as a default would bind the original value of this variable to the function.
3. It seems that author of `resrved_tokens` thought the same way.

If you are sure, I am ready to change it to `_ALPHANUMERIC_CHAR_SET`.